tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
+    "target": "ESNext",
```

Comment:
Do we need this?

Reply:
That's when I tried to iterate over Object.entries when experimenting earlier. Is it unnecessary?

Reply:
What warnings/errors are you getting without this?

Reply:
That the iteration above is only for target: ES2015 above or something.